Directory.Build.props

```diff
@@ -86,7 +86,7 @@
     <MicrosoftGraphVersion>4.36.0</MicrosoftGraphVersion>
     <MicrosoftGraphBetaVersion>4.57.0-preview</MicrosoftGraphBetaVersion>
     <MicrosoftExtensionsHttpVersion>3.1.3</MicrosoftExtensionsHttpVersion>
-    <MicrosoftIdentityAbstractions>5.0.0</MicrosoftIdentityAbstractions>    
+    <MicrosoftIdentityAbstractions>5.1.0</MicrosoftIdentityAbstractions>    
```

Comment:
make sure to update w/main branch often, but with the release on Thursday, a few of these were updated.

Reply:
done